apps/src/applab/commands.js

```diff
@@ -434,42 +436,34 @@ applabCommands.getDirection = function(opts) {
 applabCommands.dot = function(opts) {
   apiValidateTypeAndRange(opts, 'dot', 'radius', opts.radius, 'number', 0.0001);
   var ctx = applabTurtle.getTurtleContext();
+  var isPenDown = Applab.turtle.isPenDown;
```

Comment:
nit: `const`?

Reply:
No need for local variable from your comment below.